src/localisation/en.json

```diff
@@ -97,7 +97,7 @@
         "errorUserDefinedFuncMsg":"User-defined function\nthis function definition contains errors",
         "errorUserDefinedVarMsg":"User-defined variable\none of the assigments for this variable contains errors",
         "EOFError": "This frame cannot be empty.",
-        "GDriveSaveFailed":"Strype coud not saved to Google Drive. Your latest changes may have not been saved, and the sync has been turned off.",
+        "GDriveSaveFailed":"Strype coud not save to Google Drive. Your latest changes may have not been saved, and the sync has been turned off.",
```

Comment:
One more correction: coud->could.  And actually, two lines up: assigments->assignments.  Maybe I should have a pass through at some point.

Reply:
If that's the logs I'm thinking about, yes I was aware of them (but unfortunately I've noticed them only after doing the merge request of the related code). So I'll keep it the related code as is for now but will change it when doing work on bug fixes pretty soon.

Reply:
Done, good point and I've tried to spot other typos in a quick pass with Word (to see the spell checker indicators).